### PR TITLE
add example requests and responses

### PR DESCRIPTION
**Pull request by VictorHarbo**
Small pr adding examples to endpoints. Please check that the JSON keys in the examples are correct

### CODE REVIEW

File: src/main/openapi/ds-storage-openapi_v1.yaml [hunk 1]
```diff
@@ -588,3 +591,59 @@ components:
...
+        data: |
+          <xip:DeliverableUnit>Metadata for DeliverableUnit</xip:DeliverableUnit>
+        cTime: 1701180992527000
+        mTime: 1701262548465000
+        cTimeHuman: "2023-11-28 15:16:32+0100"
+        mTimeHuman: "2023-11-29 13:55:48+0100"
+        parentId: "null"
+        childrenIds: [ "ds.radio:oai:man:b39d4884-66d2-4e49-b164-0d5c7e05ca49" ]
+        parent: "null"
+        children: "<xip:Manifestation>Metadata for Manifestation</xip:Manifestation"
```
Comment (thomasegense) [inline src/main/openapi/ds-storage-openapi_v1.yaml:628]: Doubt this is correct. The child(array) is a again list of full records and not just the metadata field.

File: src/main/openapi/ds-storage-openapi_v1.yaml [hunk 2]
```diff
@@ -588,3 +591,59 @@ components:
...
+
+    GetDSRecordWithChildren:
+      value:
+        id: "ds.radio:oai:du:c124a602-7fe6-470e-a484-c771db44aee1"
+        orgid: "null"
+        id_error: false
+        origin: "ds.radio"
+        recordType: "DELIVERABLEUNIT"
+        deleted: false
+        data: |
```
Comment (thomasegense) [inline src/main/openapi/ds-storage-openapi_v1.yaml:619]: why the pipe '|' character ?

**Review by thomasegense — CHANGES_REQUESTED** [review]
A few fixes  or answers request

File: src/main/openapi/ds-storage-openapi_v1.yaml (hunk 1 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-storage-openapi_v1.yaml:628]: You are correct! I will fix this

File: src/main/openapi/ds-storage-openapi_v1.yaml (hunk 2 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-storage-openapi_v1.yaml:619]: It's not needed here, but its used to preserve the format of the following lines directly as written

**Review by thomasegense — APPROVED** [review]
looks correct now

**Pull request merged**